src/main/java/com/jlr/ttl/poc/loadmanager/models/Load.java

```diff
@@ -16,6 +19,11 @@ public class Load {
     @Column
     private String status;
 
+    @Column
+    private Carrier carrier;
+
+    @Column(name = "end_date")
+    private Date date;
 
     public Load() {}
```

Comment:
Remove constructors and getters and setters. Use the following annotations instead:

AllArgsConstructor
NoArgsConstrutctor
Data
Build